WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AddProductFeatures/AddProductFeaturesView.swift

```diff
@@ -57,7 +57,7 @@ struct AddProductFeaturesView: View {
                 }
 
                 Button(action: {
-                    // TODO: show tone bottom sheet
+                    viewModel.onSetToneAndVoice()
```

Comment:
Should we present the AI tone sheet in this view and set `presentationDetents` if the device is using iOS 16 like what we did with generating the product name? That simplifies the code for the container view, and we can clean up after dropping iOS 15.

Reply:
Done in ac96d081c9186ce060ddb4c05ed7768b8ce01648 and db17ef9bfabab40491e9e605eda12bd3f5c4618a

Reply:
@selanthiraiyan I think we should present the sheet directly in `AIToneVoiceView` without the need to go through the view model to the container view. That could be simpler. Is there a reason using the container is better that I'm missing?